src/firebolt/model/database.py

```diff
@@ -121,3 +125,40 @@ def delete(self) -> Database:
         return Database.parse_obj_with_service(
             response.json()["database"], self._service
         )
+
+    def update(self) -> Database:
+        """
+        Updates a database from Firebolt. Parameters, that will be updated: description
+        """
+
+        class _DatabaseUpdateRequest(FireboltBaseModel):
+            """Helper model for sending Database creation requests."""
+
+            account_id: str
+            database: Database
+            database_id: str
+            update_mask: FieldMask
+
+        logger.info(
+            f"Updating Database (database_id={self.database_id}, "
+            f"name={self.name}, description={self.description})"
+        )
+
+        payload = _DatabaseUpdateRequest(
```

Comment:
Do you think we can get rid of `_DatabaseUpdateRequest` and just create a plain dict here?
Do we have any profit of using it?

Reply:
It just seems that it's redundant here